getting_hired/project_portfolio.md

```diff
@@ -14,6 +14,38 @@ Remember to write some documentation for both technical and non-technical audien
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
 
+### Design Inspirations
+<details markdown="block">
+  <summary> Links for designs </summary>
```

Comment:
"Show Designs" in conjunction with the introduction is more appropriate.

Reply:
I have made requested changes.